### ISSUE TITLE
Can it be used with beacon? What are other usecases.

### ISSUE THREAD

**Issue by ahmed1490** [joshblour/discovery-react#1]
The readme isnt really clear to me.

**Comment by yonahforst** [comment]
I'll try to write some more in the readme.

The basic idea is that it turns your device into a "beacon" broadcasting a short ID so that other users nearby can discovery you (and visa-versa)

The usecases all revolve around making the app aware of nearby users, even if those users are running the app in the background. Maybe it's a game and you want users to play with other users on the same bus or metro.

The beacon scanning/transmitting built into iOS works a bit differently. You can register for a beacon region and then get woken up when you enter/exit it. But you can only advertise as a beacon from the foreground.

In other words, transmitting as an iBeacon lets you notify **other** background users that **you** are nearby but can't tell you who those other nearby users are.

This library will tell you which other users are nearby (even if they are in the background) but it can't wake **them** up to tell them that **you** are nearby.

sadly, you can't use the library _and_ transmit as an iBeacon at the same time.

oh, this is also cross-platform. so you can discover android users from iOS and visa-versa
